### DISCUSSION TITLE
Welcome to cosmos-sdk Discussions!

### DISCUSSION THREAD

**Discussion by KOSASIH** [KOSASIH/cosmos-sdk#677] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MAhdi3169** [comment]
Mr.kosasih

I emailed you, but did not receive a response. I left a comment,but did not receive a response.

Please respond to me. I have a question for you.

Do you actually collaborate with the Pi Network team?

Please, if you value your work and professionalism, answer me.

I am analyzing your work and will present it to the pioneers in Iran so they know what you are doing. After seeing your coding, they have become fans of yours.

So, please, respond to me.

**Comment by MAhdi3169** [comment]
![Screenshot_۲۰۲۵-۰۹-۰۹-۰۲-۳۹-۳۳-۷۴۹_com.google.android.gm.jpg](https://github.com/user-attachments/assets/c84bc5aa-2ebe-4865-af0f-3042b8ef7f2d)
